src/business-logic/services/ingested-entity-to-entity-mapper.ts

```diff
@@ -52,6 +52,8 @@ export class IngestedEntityToEntityMapper {
       rundownId: ingestedSegment.rundownId,
       name: ingestedSegment.name,
       rank: ingestedSegment.rank,
+      isHidden: ingestedSegment.isHidden,
```

Comment:
It's basically whether and not to show it. It's possible for a Segment to be perfectly "valid" and "executable", but still shouldn't be shown nor included in the execution (floating a Segment in iNews for example).

Reply:
It could also be `isIgnored`. I am not fan of it, since it doesn't properly describe what its purpose is in the backend.
I'll leave it be for now, but I think we should consider a better name for it.